test/font_test.py

```diff
@@ -542,6 +536,7 @@ def test_load_from_file_bytes(self):
             os.path.split(pygame.__file__)[0], pygame_font.get_default_font()
         )
         filesystem_encoding = sys.getfilesystemencoding()
+        filesystem_errors = "replace" if sys.platform == "win32" else "surrogateescape"
```

Comment:
where did this come from?

Reply:
It was in the `compat.py`. It is a function that, once removed py3 alternatives, does essentially what you see in that line, so I replaced the only case I saw in the project with the code and deleted the function from `compat.py` (and eventually deleted the file after it was empty).

Reply:
Yeah, but the diff doesn't show it replacing anything, just popping up out of nowhere.